pkg/tenant/apikeys.go

```diff
@@ -65,13 +66,9 @@ func (s *Server) ProjectAPIKeyList(c *gin.Context) {
 	}
 	for _, key := range reply.APIKeys {
 		out.APIKeys = append(out.APIKeys, &api.APIKey{
-			ID:          key.ID.String(),
-			ClientID:    key.ClientID,
-			Name:        key.Name,
-			Owner:       key.CreatedBy.String(),
-			Permissions: key.Permissions,
-			Created:     key.Created.Format(time.RFC3339Nano),
-			Modified:    key.Modified.Format(time.RFC3339Nano),
+			ID:       key.ID.String(),
+			ClientID: key.ClientID,
+			Name:     key.Name,
```

Comment:
List is only supposed to return some of the fields (see the quarterdeck implementation) so I've updated the handler to reflect that.